### ISSUE TITLE
[Bug] Colored Text not working

### ISSUE THREAD

**Issue by ApexModder** [rauenzi/BetterDiscordApp#533]
**Describe the Bug**
Colored Text setting to color users messages to same color as their respective role does not work when enabled.

Note: This is a repost from an [issue I posted on the BD Installer](https://github.com/rauenzi/BBDInstaller/issues/20) repo by mistake, thought I was on this repo when I opened the issue

**To Reproduce**
Enable Colored Text in BetterDiscord's Settings
Goto server with roles and see text is not colored

**Screenshots**
[![](https://i.imgur.com/G4wVpWcm.png)](https://i.imgur.com/G4wVpWc.png)
[![](https://i.imgur.com/dlXWyf4m.png)](https://i.imgur.com/dlXWyf4.png)

**Discord Version**
```
Stable 74008 (d6b7151)
BBD 0.3.5
Host 0.0.309
Injector 0.6.2
Windows 10 64-Bit (10.0.19042)
```

**Additional Context**
I have the following plugins installed, a few of them also modify text coloring based on roles but this issue happens even if i disable all of my plugins.
```
AccountDetailsPlus v0.1.10
BDFDB v1.2.6
BetterRoleColors v0.8.13
BetterTypingUsers v1.0.2
PluginRepo v2.1.1
ServerSearch v0.1.4
ThemeRepo v2.1.1
TypingIndicator v0.4.3
VoiceEvents v1.3.3
ZeresPluginLibrary v1.2.27
```

**Comment by Strencher** [comment]
PR's to fix this issue can be found at #531 & #530.

**Comment by rauenzi** [comment]
PR was merged.
